### ISSUE TITLE
Unable to create a dataset without resources

### ISSUE THREAD

**Issue by madelinekosse** [ckan/ckanext-scheming#377]
I have set `CKAN__DATASET__CREATE_ON_UI_REQUIRES_RESOURCES=False` in my ckan configuration, as I wish to create datasets without any resources. However, the addition of the scheming plugin has removed that functionality. In the scheming "add dataset" page, a draft dataset is created at the end of each step and it is only converted to an active state when a resource is added. My understanding is that the post to the create dataset endpoint is intercepted to add the draft state, and then the active state is set in the plugin's new resource handler. The result is that if I click through the new dataset page and click "finish" without adding a resource to the resource page, nothing happens. I believe this is a bug, as we should be able to finish creating the dataset without adding a new resource.

**Comment by wardi** [comment]
yes, sounds like a bug. ckanext-scheming only overrides the templates but not the views so there's probably a difference in the core templates that needs to be copied into ckanext-scheming.

**Comment by madelinekosse** [comment]
I believe it is [this line](https://github.com/ckan/ckan/blob/904d8183caf864dc9981c3f3ce5100e125c3cc3f/ckan/views/dataset.py#L571) that needs to be included in [this function](https://github.com/ckan/ckanext-scheming/blob/3046c055513c087ca8a0404cf652d2422f7fd0c3/ckanext/scheming/views.py#L158)
